src/app/page.tsx

```diff
@@ -1,6 +1,15 @@
 import Image from "next/image";
+import { useEffect, useState } from "react";
 
 export default function Home() {
+  const [isDark, setIsDark] = useState(false);
+
+  useEffect(() => {
+    if (true) {
+      setIsDark(true);
+    }
+  }, [isDark]);
```

Comment:
anthropic review: The `useEffect` hook has an infinite loop due to `isDark` being in the dependency array while also being set inside the effect.